docker-compose.yaml

```diff
@@ -1,5 +1,20 @@
-version: "1"
+version: "3"
+
+networks:
+  default:
+    name: espresso-sequencer
+
 services:
+  demo-l1-network:
```

Comment:
Can we actually name this `l1` to be consistent with what we have in the OP repo?